docs/INTEGRATION.md

```diff
@@ -9,24 +13,24 @@
 To run headless Cypress tests, enter the following command from the root of the project:
 
 ```shell
-yarn test-cypress
+yarn test-Cypress
```

Comment:
`error Command "test-Cypress" not found. Did you mean "test-cypress"?`

Reply:
Bad find+replace from me. Will fix